pkg/loki/loki.go

```diff
@@ -305,7 +305,7 @@ type Loki struct {
 	runtimeConfig             *runtimeconfig.Manager
 	MemberlistKV              *memberlist.KVInitService
 	compactor                 *compactor.Compactor
-	QueryFrontEndTripperware  queryrangebase.Tripperware
+	QueryFrontEndTripperware  queryrangebase.Middleware
```

Comment:
We should consider renaming this member to avoid confusion

Reply:
Yes. Same for all factory methods.

Reply:
Done.